### PR TITLE
mysql

### PR DESCRIPTION
**Pull request by g0djan**
Support of MySQL

Next steps:
- Update Readme.md(split it by pg and mysql)
- Split logic of wal-g and postgres

### CODE REVIEW

File: docker/mysql_tests/scripts/tests/binlog_push_test.sh [hunk 1]
```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+set -e -x
+
+export WALE_S3_PREFIX=s3://mysqlbinlogpushbucket
+export WALG_MYSQL_DATASOURCE_NAME=sbtest:@/sbtest
+export WALG_MYSQL_BINLOG_DST=${MYSQLDATA}
+
+echo "CREATE DATABASE sbtest;" > /etc/mysql/init.sql
```
Comment (Tinsane) [inline docker/mysql_tests/scripts/tests/binlog_push_test.sh]: Let's extract it to a separate file, which will be delivered to the container via Dockerfile COPY.

File: docker/mysql_tests/scripts/tests/binlog_push_test.sh [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+echo "CREATE USER sbtest@localhost;" >> /etc/mysql/init.sql
+echo "GRANT ALL PRIVILEGES ON *.* TO sbtest@localhost;" >> /etc/mysql/init.sql
+
+echo "[mysqld]" >> /etc/mysql/my.cnf
```
Comment (Tinsane) [inline docker/mysql_tests/scripts/tests/binlog_push_test.sh]: Let's extract it to a separate file, which will be delivered to the container via Dockerfile COPY.

File: docker/mysql_tests/scripts/tests/delete_end_to_end_test.sh [hunk 3]
```diff
@@ -0,0 +1,69 @@
+#!/bin/sh
+set -e -x
+
+export WALE_S3_PREFIX=s3://mysqldeleteendtoendbucket
+export WALG_MYSQL_DATASOURCE_NAME=sbtest:@/sbtest
+export WALG_MYSQL_BINLOG_DST=${MYSQLDATA}
+
+echo "CREATE DATABASE sbtest;" > /etc/mysql/init.sql
```
Comment (Tinsane) [inline docker/mysql_tests/scripts/tests/delete_end_to_end_test.sh]: Let's extract it to a separate file, which will be delivered to the container via Dockerfile COPY.

File: docker/mysql_tests/scripts/tests/delete_end_to_end_test.sh [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+echo "CREATE USER sbtest@localhost;" >> /etc/mysql/init.sql
+echo "GRANT ALL PRIVILEGES ON *.* TO sbtest@localhost;" >> /etc/mysql/init.sql
+
+echo "[mysqld]" >> /etc/mysql/my.cnf
```
Comment (Tinsane) [inline docker/mysql_tests/scripts/tests/delete_end_to_end_test.sh]: Let's extract it to a separate file, which will be delivered to the container via Dockerfile COPY.

**Comment by g0djan** [comment]
@Tinsane

**Pull request merged**